authors/apps/comments/views.py

```diff
@@ -0,0 +1,143 @@
+from django.shortcuts import get_object_or_404
+
+from rest_framework import generics
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+
+from . import serializers
+from .models import Comment
+from authors.apps.articles.models import Article
+
+
+class CreateCommentAPiView(generics.ListCreateAPIView):
+    permission_classes = (IsAuthenticated,)
+
+    queryset = Comment.objects.all()
+
+    def post(self, request, *args, **kwargs):
+        slug = self.kwargs['slug']
+        try:
+            article = Article.objects.get(slug=slug)
+        except Article.DoesNotExist:
+            message = 'Article does not exist'
+            return Response(message, status.HTTP_404_NOT_FOUND)
+        comment = request.data
+        comment.update({'author': request.user.pk, 'article': article.id})
+        serializer = serializers.CommentSerializer(data=comment)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+        return Response({"message": "Comment Successfully added"},
+                        status=status.HTTP_201_CREATED)
+
+    def get(self, *args, **kwargs):
+        slug = self.kwargs['slug']
+        try:
+            article = Article.objects.get(slug=slug)
+        except Article.DoesNotExist:
+            message = 'Article does not exist'
```

Comment:
@Opio-Emmanuel-Omona Some of these messages seem to appear multiple times. Could you avoid repeating yourself.